src/api/export.ts

```diff
@@ -18,7 +18,8 @@ type Opts = {
 	cwd?: string,
 	static?: string,
 	basepath?: string,
-	host_header?: string,
+  host_header?: string,
```

Comment:
Looks like i'm using the wrong number of spaces or tabs, indentation looked fine in my editor.
I'll clean it up and `git push -f` the fixed commit.